### PR TITLE
do not automatically download lfs files when cloning/checking out

### PR DESCRIPTION
**Pull request by diosmosis**
See https://github.com/git-lfs/git-lfs/issues/2717

### CODE REVIEW

**Comment by diosmosis** [comment]
Need to modify travis-scripts to do a git lfs pull before merging this.

**Comment by mattab** [comment]
* Update https://developer.matomo.org/guides/tests-ui and possible other guides which may need updating

File: .travis.yml [hunk 1]
```diff
@@ -86,6 +86,7 @@ before_install:
 
 install:
   - git fetch -q
+  - git lfs pull
```
Comment (sgiehl) [inline .travis.yml]: Any reason for adding that here globally? Wouldn't it be enough to add that to `tests\travis\install_git_lfs.sh`
Guess there is no use in doing that for non UI tests.

**Review by sgiehl — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (diosmosis) [reply .travis.yml]: Updated.

**Comment by sgiehl** [comment]
UI tests are still all failing. Seems the images are not downloaded correctly. Haven't tested, but might the order of `git lfs fetch`, `checkout` and `pull` be wrong?

**Comment by diosmosis** [comment]
@sgiehl build is working now

**Review by sgiehl — APPROVED** [review]

**Comment by diosmosis** [comment]
@mattab created simple dev-docs PR here: https://github.com/matomo-org/developer-documentation/pull/265

**Pull request merged**